jerry-libc/jerry-libc.c

```diff
@@ -284,5 +260,8 @@ rand (void)
 void
 srand (unsigned int seed) /**< new seed */
 {
+  libc_random_gen_state[0] =
+  libc_random_gen_state[1] =
+  libc_random_gen_state[2] =
```

Comment:
Most probably it is increasing the code size with some bytes. However, I cannot think of a smaller solution for fixing `srand`. Since that function _must_ reinitialize the whole state of the pseudo-RNG to ensure that always the same sequence is generated from the same seed.